Framework/CurveFitting/src/Algorithms/CrystalFieldEnergies.cpp

```diff
@@ -46,7 +46,7 @@ const std::string CrystalFieldEnergies::summary() const {
 void CrystalFieldEnergies::init() {
 
   // Input
-  auto bounds = boost::make_shared<Kernel::BoundedValidator<int>>(1, 13);
+  auto bounds = boost::make_shared<Kernel::BoundedValidator<int>>(-99, 13);
   declareProperty("Nre", 1, bounds, "A rare earth ion. Possible values are: "
```

Comment:
Yes. I guess we can refactor the code in `CrystalElectricField` to use a string instead. And then we can remove all instances of `nre`. I just implemented as it is now because it was the easiest way to go... but I can change it.